package.json

```diff
@@ -56,10 +56,10 @@
     "joi": "^17.7.0",
     "js-yaml": "^4.1.0",
     "keymbinatorial": "^2.0.0",
-    "screwdriver-data-schema": "^23.5.1",
+    "screwdriver-data-schema": "file:../data-schema/screwdriver-data-schema-22.0.0.tgz",
```

Comment:
TODO: Update the version after https://github.com/screwdriver-cd/data-schema/pull/575 is merged